### DISCUSSION TITLE
How do I know what API & model was used by default for any generated output?

### DISCUSSION THREAD

**Discussion by itsRevela** [danielmiessler/Fabric#383] [Q&A]
- How do I know what api & model is being utilized without specifying the model when running a pattern?
- If I do use a pattern, are there logs or debug messages available that describe what model is being used, or what's going on in the backend, etc?
- For reference, I have included in my Fabric set-up keys for all 3 supported APIs (OpenAI/GoogleYouTube/Anthropic)

**Comment by digitalw00t** [comment]
DO this:
grep DEFAULT_MODEL ~/.config/fabric/.env

It'll tell you what your default is.  If you didn't specify a model that is it.  Otherwise you'll have to create a session, and the information is stored in the ~/.config/fabric/sessions/<SESSIONNAME> file that's stored there.
